assets/benchmarks/shaders/Benchmark_VsSimpleQuads.hlsl

```diff
@@ -14,9 +14,18 @@
 
 #include "VsOutput.hlsli"
 
-VSOutputPos vsmain(float4 Position : POSITION)
-{
-	VSOutputPos result;
-	result.position = Position;
-	return result;
+float randomCompute(uint32_t instCount, float4 Position) {
+  float ret = frac(sin(Position.x) + cos(Position.y));
```

Comment:
sorry i forget to mention, try to avoid using EFU (cos, sin) instructions if possible, they are special instructions that are more costly. My worry (i could be wrong though) is that those instructions might consume more power than normal instructions